core/src/main/scala/org/apache/spark/api/python/PythonRDD.scala

```diff
@@ -313,6 +313,7 @@ private object SpecialLengths {
   val PYTHON_EXCEPTION_THROWN = -2
   val TIMING_DATA = -3
   val END_OF_STREAM = -4
+  val NULL = -5
```

Comment:
So what was the discussion here? Before, the motivation was that Nones from Python would be encoded as a pickled version of None. Is it that we now want to send nulls back, and we are not using the pickle library for Java that would map them? Then it would be okay.